### DISCUSSION TITLE
Security Question - Remote IPs

### DISCUSSION THREAD

**Discussion by afyles** [DaxStudio/DaxStudio#1315] [General]
@dgosbell My IT Sec team is complaining that the software reaches out to a few IPs outside the US and therefore doesn't want to allow us to continue to use DaxStudio.  Is this something you are aware of and could provide any context on?

**Comment by dgosbell** [comment]
It would be one of the 4 types of requests in the "Web Requests" section here https://daxstudio.org/docs/security-privacy/#web-requests

We have these settings to control these https://daxstudio.org/docs/features/privacy-settings/ and it's possible to turn them all off. If there are specific IPs they are worried about I'd be interested to know. The above are all different cloud services so I'm not specifically doing any deliberate region routing.

**Comment by afyles** [comment]
Turning those off will help.  They found IPs to the Netherlands and
Russia.

**Comment by marcosqlbi** [comment]
@afyles do you have more details about these IPs/calls? They don't seem generated by DAX Studio.

**Reply by afyles** [reply]
Going to get the team to update this thread
